.storybook/webpack.config.js

```diff
@@ -1,30 +1,28 @@
-"use strict";
-
 const path = require("path");
 const webpack = require("webpack");
 
 const getConfig = require("../config/config").getConfig;
 
 module.exports = {
   resolve: {
-    alias: {
-      "devtools": path.join(__dirname, "./public/js/lib/devtools"),
-      "devtools-sham": path.join(__dirname, "./public/js/lib/devtools-sham"),
-      "sdk": path.join(__dirname, "./public/js/lib/devtools-sham/sdk")
+      alias: {
```

Comment:
Why all these changes? Looks like the indentation just got messed up, unless I'm missing something.